src/contour/ui.template/main.qml.in

```diff
@@ -39,11 +39,13 @@ ApplicationWindow
     onWidthChanged : function() {
         vtui.width = width
         vtui.session.requestWindowResize(width, height)
+        vtui.updateSizeWidget()
     }
 
     onHeightChanged: function() {
         vtui.height = height
         vtui.session.requestWindowResize(width, height)
+        vtui.updateSizeWidget()
```

Comment:
One last thing (apart from changelog(/docs/)) missing IMHO is, that this should not be shown when the window is resized but the display grid resolution remains the same.

So instead of hooking into onWidthChanged/onHeightChanged, it probably makes most sense to hook into `vtWidget.session.pageColumnsCount` (and pageLinesCount` updates.

Reply:
from discussion in discord we decided to make separate PR to update it